lib/kitchen/driver/docker.rb

```diff
@@ -212,7 +213,16 @@ def parse_image_id(output)
       def build_image(state)
         cmd = "build"
         cmd << " --no-cache" unless config[:use_cache]
-        output = docker_command("#{cmd} -", :input => dockerfile)
+        # Previously we sent the Dockerfile to stdin, which works fine until you want
+        # to use things like COPY or ADD in a Dockerfile.
+        # To workaround this, write out Dockerfile to a tmp file in the current dir,
+        # and remove after running docker.
+        # Needs to be a unique filename, as we may have concurrent test-kitchen runs.
+        output = Tempfile.create('Dockerfile-kitchen-', Dir.pwd) do |tempf|
```

Comment:
Didn't know `create()` was a method, alias for `new()`? Why not just use system tmp dir instead of $PWD?